README.md

```
@@ -1,54 +1,108 @@
+### Linting & Type Checking
+
+```sh
+npm run lint       # ESLint check
+npm run lint:fix   # ESLint auto-fix
+npm run typecheck  # TypeScript type check (no emit)
+npm run knip       # Detect unused files and exports
+```
+
+## Project Structure
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)